api/src/search/backend/load_opportunities_to_index.py

```diff
@@ -66,13 +66,43 @@ def __init__(
         self.set_metrics({"index_name": self.index_name})
 
     def run_task(self) -> None:
+        logger.info("Creating multi-attachment pipeline")
+        self._create_multi_attachment_pipeline()
         if self.is_full_refresh:
             logger.info("Running full refresh")
             self.full_refresh()
         else:
             logger.info("Running incremental load")
             self.incremental_updates_and_deletes()
 
+    def _create_multi_attachment_pipeline(self) -> None:
+        """
+        Create multi-attachment processor
+        """
+        pipeline = {
+            "description": "Extract attachment information",
+            "processors": [
+                {
+                    "foreach": {
+                        "field": "attachments",
+                        "processor": {
+                            "attachment": {
+                                "target_field": "_ingest._value.attachment",
+                                "field": "_ingest._value.data",
+                            }
+                        },
+                    }
+                }
+            ],
+        }
+        pipeline_name = "multi-attachment"
+        resp = self.search_client._client.ingest.put_pipeline(id=pipeline_name, body=pipeline)
+        # Check the response
+        if resp["acknowledged"]:
+            logger.info(f"Pipeline '{pipeline_name}' created successfully!")
+        else:
+            logger.error(f"Error creating pipeline: {resp}")
```

Comment:
I think making a method in our search client called `put_pipeline` would be useful for the future.